charts/lily/templates/notifier/app.yaml

```diff
@@ -0,0 +1,109 @@
+kind: StatefulSet
+metadata:
+  name: {{ .Release.Name }}-lily
+  labels:
+    {{- include "sentinel-lily.allLabels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ .Release.Name }}-lily-daemon-service
+  podManagementPolicy: {{ .Values.podManagementPolicy | default "Parallel" | quote }}
+  selector:
+    matchLabels:
+      {{- include "sentinel-lily.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "sentinel-lily.allLabels" . | nindent 8 }}
+    spec:
+      {{- include "sentinel-lily.common-lily-affinity-selectors" . | indent 6 }}
+      imagePullSecrets:
+      - name: regcred
+      initContainers:
+      - name: init-datastore
+        image: {{ include "sentinel-lily.docker-image" . | quote }}
+        imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+        command: ["/bin/sh", "-c"]
+        args:
+        {{- include "sentinel-lily.initialize-datastore-script" . | nindent 10 }}
+        env:
+        {{- include "sentinel-lily.common-envvars" . | indent 8 }}
+        volumeMounts:
+        {{- include "sentinel-lily.common-volume-mounts" . | indent 8 }}
+        resources:
+          # resources required to initialize the datastore are small
+          {{- (include "sentinel-lily.minimal-resources" .) | indent 10 }}
+      containers:
+      - name: daemon
+        image: {{ include "sentinel-lily.docker-image" . | quote }}
+        imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+        command: ["lily"]
+        args:
+        - daemon
+        {{- range .Values.daemon.args }}
+        - {{ . }}
+        {{- end }}
+        env:
+        {{- include "sentinel-lily.common-envvars" . | indent 8 }}
+        ports:
+        - containerPort: 1234
+          name: api
+        - containerPort: 1347
+          name: p2p
+        {{- if .Values.prometheusOperatorServiceMonitor }}
+        - containerPort: 9991
+          name: metrics
+        {{- end }}
+        volumeMounts:
+        {{- include "sentinel-lily.common-volume-mounts" . | indent 8 }}
+        lifecycle:
+          postStart:
+            exec:
+              command:
+                - "/bin/sh"
+                - "-c"
+                  # lifecycle.postStart.exec.command doesn't accept args
+                  # so we execute this script as a multiline string
+                {{- include "sentinel-lily.common-job-start-script" . | indent 16 }}
+        resources:
+        {{- if .Values.cluster.notifier.resources }}
+          {{- toYaml .Values.cluster.notifier.resources | nindent 10 }}
+        {{- else }}
+          # Required resources to run daemon lily are signficant and the below default
+          # may be insufficient. Please see https://lilium.sh/software/lily/hardware/
+          # for more information.
+          requests:
+            cpu: "8000m"
+            memory: "64Gi"
+          limits:
+            cpu: "8000m"
+            memory: "64Gi"
```

Comment:
This changes the notifier from 256Gi RAM to 64Gi ram, correct? Looks good if so.